flake.nix

```diff
@@ -14,46 +14,41 @@
       system = "x86_64-linux";
       pkgs = import nixpkgs {
         inherit system; config.allowUnfree = true;
-        overlays = builtins.attrValues self.overlays;
```

Comment:
When removing overlay, the redpanda in devshell is referencing to broken nixpkgs version. Maybe remove redpanda form devshell? Probably do not need it there